brayns/Brayns.cpp

```diff
@@ -143,12 +144,20 @@ struct Brayns::Impl : public PluginAPI
                                  << "' is not a valid Brayns plugin; missing "
                                     "brayns_plugin_create()"
                                  << std::endl;
-                    return;
+                    exit(0);
```

Comment:
EXIT_FAILURE

Reply:
(...or throw to have a single exit point below)